tree/ntuple/v7/inc/ROOT/RColumnElement.hxx

```diff
@@ -692,14 +692,14 @@ public:
 /// Tightly packs `count` floats contained in `src` into `dst` using `nFloatBits` per float.
 /// `nFloatBits` must be >= kReal32TruncBitsMin and <= kReal32TruncBitsMax.
 /// The extra bits are dropped from the mantissa. The sign and exponent bits are always preserved.
-/// IMPORTANT: the size of `dst` must be rounded up from `count * nFloatBits`
-/// to the next multiple of 4 (i.e. the word size).
+/// IMPORTANT: the size of `dst` must be at least `(count * nFloatBits + 63) / 64` (i.e.
+/// it needs to be rounded up to the next word).
```

Comment:
Is this a strict requirement or could the contents also be packed at byte granularity?

Reply:
Do you have measured evidence that it's actually faster? If all writes go to cache, it might not matter at all. Also, this should only be important for the last word which requires special handling anyway, right? There's also another problem that many platforms (ARM, PPC) require alignment for 4 and 8 byte reads, which might be hard to guarantee if we ever `mmap` pages into memory.

Reply:
Turns out that rewriting the packing code to byte granularity also significantly simplified it :)